treeherder/etl/common.py

```diff
@@ -123,3 +126,27 @@ def get_guid_root(guid):
 def to_timestamp(datestr):
     """Converts a date string to a UTC timestamp"""
     return calendar.timegm(parser.parse(datestr).utctimetuple())
+
+
+def get_end_of_day(date):
+    new_date = datetime.datetime.strptime(date, '%Y-%m-%d') + datetime.timedelta(days=1, microseconds=-1)
+    return new_date.strftime('%Y-%m-%d %H:%M:%S.%f')
+
+
+def get_tree(param):
+    if param == 'trunk':
+        # the repository id's are used instead of name for query efficiency;
+        # trunk equals mozilla-central', 'mozilla-inbound', 'autoland', 'fx-team' (in sequence)
+        tree = (1, 2, 77, 14)
```

Comment:
I must admit my knee-jerk reaction here is "no magic numbers" and that we should fetch these ids from the db.  But if Ed feels like this is ok, then I'm cool with it, too.  :)

Reply:
I get that part of the argument for using these hard numbers is that they're specified in the fixture already, so we know they're consistent across all deployments.

Reply:
Also curious: Why a tuple here, rather than an array like for the other cases below?

Reply:
I started out with a tuple initially when using just the tree names, and I just kept it at that since I didn't think we'd be adding to it within that function. I can change it to a list if you think that'd make it more versatile. :)